cli/src/main/java/com/box/l10n/mojito/cli/command/CommitToPullRunCommand.java

```diff
@@ -0,0 +1,87 @@
+package com.box.l10n.mojito.cli.command;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.command.param.Param;
+import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.json.ObjectMapper;
+import com.box.l10n.mojito.rest.client.CommitClient;
+import com.box.l10n.mojito.rest.entity.Repository;
+import org.fusesource.jansi.Ansi.Color;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * @author jaurambault
+ */
+@Component
+@Scope("prototype")
+@Parameters(commandNames = {"commit-to-pull-run"}, commandDescription = "Associate a commit to a Pull Run")
+public class CommitToPullRunCommand extends Command {
+
+    /**
+     * logger
+     */
+    static Logger logger = LoggerFactory.getLogger(CommitToPullRunCommand.class);
+
+    @Autowired
+    ConsoleWriter consoleWriter;
+
+    @Parameter(names = {Param.REPOSITORY_LONG, Param.REPOSITORY_SHORT}, arity = 1, required = true, description = Param.REPOSITORY_DESCRIPTION)
+    String repositoryParam;
+
+    @Parameter(names = {Param.COMMIT_HASH_LONG, Param.COMMIT_HASH_SHORT}, arity = 1, required = true, description = "The commit to map to a pull run")
+    String commitHash;
+
+    @Parameter(names = {"--input-directory", "-i"}, arity = 1, required = false, description = "The input directory where to read the pull run name, filename: pull-run-name.txt)")
```

Comment:
Should we allow the full path of the file to be specified instead of just the input dir? (--input-file-path)

Reply:
that was my original intent but then i wanted to avoid having too many parameter until proven necessary. For our current usages, the file will be collocated with the localized files which I think is fine. For a repository where we'd scan from the root it would put that file in a place we probably don't want to have.